app/controllers/prison/visits_controller.rb

```diff
@@ -4,19 +4,22 @@ class Prison::VisitsController < ApplicationController
   before_action :authenticate_user
   before_action :cancellation_reason_set, only: :cancel
   before_action :visit_is_processable, only: %i[process_visit update]
+  after_action  :track_visit_process, only: :update
```

Comment:
We call the method inside `#update` when the booking response is a success so I think we can remove this after action.